template/.polarion/tracker/fields/workitem-link-role-enum.xml

```diff
@@ -130,6 +130,8 @@ SPDX-License-Identifier: Apache-2.0 -->
     </option>
     <option id="exchanges" description="The exchanges connecting the Function." name="exchange" oppositeName="is exchange on" sortOrder="110">
         <rule from="systemFunction, logicalFunction, physicalFunction" to="functionalExchange" />
+        <rule from="componentExchange" to="functionalExchange" />
```

Comment:
This is done via `allocated_functional_exchanges`, isn't it?
But we should add the link from the physicalComponent types to the componentExchange (via their ports)
```suggestion
        <rule from="physicalComponent" to="componentExchange" />
        <rule from="physicalComponentNode" to="componentExchange" />
        <rule from="physicalComponentBehavior" to="componentExchange" />
        <rule from="physicalActor" to="componentExchange" />
```